src/studio/src/designer/backend.Tests/FluentTestsBase.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Designer.Tests;
+
+/// <summary>
+/// Base class for fluent tests.
+/// </summary>
+/// <typeparam name="T"></typeparam>
+public class FluentTestsBase<T>
```

Comment:
Note for reviewer: This class should eventually be moved to some common nuget package that should be used across all projects.